### PR TITLE
Resolve issue #318: Wrong log info about executing npm process

### PR DESCRIPTION
**Pull request by huanlin**
- Fixed Log content.
- Exclude Deploy folder from Build.csproj.
- Removed invalid items from csproj file.

### CODE REVIEW

File: src/FlubuCore/Tasks/Process/RunProgramTask.cs [hunk 1]
```diff
@@ -250,7 +250,14 @@ protected override int DoExecute(ITaskContextInternal context)
 
             DoLogInfo($"Running program from '{workingFolder}':");
 
-            DoLogInfo($"{command.CommandName} {commandArgs}{Environment.NewLine}", Color.DarkCyan);
+            if (command.CommandName.Contains("cmd.exe"))
```
Comment (huanlin) [inline src/FlubuCore/Tasks/Process/RunProgramTask.cs:253]: I've tested with `npm` and `choco`, and this is how I try to satisfy both cases while maintaining command arguments masks work.

**Comment by mzorec** [comment]
I've noticed some tests are failling in FlubuCore.Tests.Tasks.RunProgramTaskTests after your changes. Could you please take a look?

**Comment by huanlin** [comment]
> I've noticed some tests are failling in FlubuCore.Tests.Tasks.RunProgramTaskTests after your changes. Could you please take a look?

Oops sorry! It should be fine now.

File: src/FlubuCore/Tasks/Process/RunProgramTask.cs (hunk 1 above)
Comment (mzorec) [reply src/FlubuCore/Tasks/Process/RunProgramTask.cs]: Maybe for loging purposes changing from:

``` c#   
 if (command.CommandName != null && command.CommandName.Contains("cmd.exe"))
 {
      DoLogInfo($"{cmd}{commandArgs}{Environment.NewLine}", Color.DarkCyan);
 }
else
 {
      DoLogInfo($"{command.CommandName}{commandArgs}{Environment.NewLine}", Color.DarkCyan);
 }
```
 to 
``` c#   
DoLogInfo($"{cmd}{commandArgs}{Environment.NewLine}", Color.DarkCyan);
```
What do you think? Not sure if we would lose any important info about the command name in the logs. I think it should be ok for all scenarios.

Comment (huanlin) [reply src/FlubuCore/Tasks/Process/RunProgramTask.cs]: Yes I think that would be OK. I assume few guys would care to know whether the program is executed via cmd.exe or not. Thanks for the suggestion!

**Pull request merged**